man/bitbutler.adoc

```diff
@@ -67,9 +67,15 @@ Work with pull requests
 
 ==== subcommands
 
+*approve* _REPO_ _PULL_REQUEST_ID_::
+Approve pull request with given id
```

Comment:
Can you add examples for these?
It might not be immediately obvious what the ID is

(See deploykeys doc for example)